frontend/server/configs.ts

```diff
@@ -124,6 +125,7 @@ export function loadConfigs(argv: string[], env: ProcessEnv): UIConfigs {
       aws: {
         accessKey: AWS_ACCESS_KEY_ID || '',
         endPoint: 's3.amazonaws.com',
+        region: AWS_REGION || 'us-east-1',
```

Comment:
Does `us-east-1` work if no region is specified? Any documentation we can add here?

Reply:
@zijianjoy Yes, minioJS is using the default S3 endpoint (s3.amazonaws.com) which is `us-east-1`